README.md

```diff
@@ -28,7 +28,19 @@ dependencies and passes them to a library called [Dependency-Check](https://gith
 
 #### Clojure CLI
 
-To install globally, add `nvd-clojure/nvd-clojure {:mvn/version "1.8.0"}`
+To install in a given project, you can add `nvd-clojure/nvd-clojure {:mvn/version "1.8.0"}` to your deps.edn.
+
+If you have CLI version 1.10.3.933 or later, you can also install `nvd-clojure` as a "tool":
```

Comment:
For a brief period, Tools usage will be an 'also' thing. Shortly thereafter we can remove all references to risky/unaccurate ways of running nvd-clojure (it was about time!).